app/services/placement-manager.js

```diff
@@ -77,7 +77,7 @@ export default Service.extend(ObjectQueryManager, {
       publisherId: get(publisher || {}, 'id'),
       templateId: get(template || {}, 'id'),
       topicId: get(topic || {}, 'id'),
-      reservePct: reservePct || 0,
+      reservePct: reservePct || null,
```

Comment:
These technically should be cast as integers. Newer versions of Apollo no longer auto-cast strings to integers, so this would actually break if we did an upgrade. I realize there's probably many other instances of this but... 🙂